scripts/dev/e2e.py

```diff
@@ -164,7 +164,8 @@ def create_test_pod(args: argparse.Namespace, dev_config: DevConfig) -> None:
                         "go",
                         "test",
                         "-v",
-                        "-timeout=60m",
+                        "-timeout=30m",
+                        "-failfast",
```

Comment:
this was added to prevent the tests from continuing when they had already failed.